integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMonitoringExporter.java

```diff
@@ -0,0 +1,385 @@
+            .command(curlDownloadCmd))
+        .execute(), "Failed to download monitoring exporter webapp");
+    String command = String.format("chmod 777 %s ", monitoringExporterBuildFile);
+    assertTrue(new Command()
+        .withParams(new CommandParams()
+            .command(command))
+        .execute(), "Failed to download monitoring exporter webapp");
+
+    command = String.format("cd %s && %s  %s/exporter/exporter-config.yaml",
+        RESULTS_ROOT,
+        monitoringExporterBuildFile,
+        RESOURCE_DIR);
+    assertTrue(new Command()
+        .withParams(new CommandParams()
+            .command(command))
+        .execute(), "Failed to build monitoring exporter webapp");
+    return RESULTS_ROOT + "/wls-exporter.war";
+  }
+
+  /**
+   * Check metrics using Prometheus.
+   *
+   * @param searchKey   - metric query expression
+   * @param expectedVal - expected metrics to search
+   * @throws Exception if command to check metrics fails
+   */
+  private static void checkMetricsViaPrometheus(String searchKey, String expectedVal, String nodeportserver)
+      throws Exception {
+
+    // url
+    String curlCmd =
+        String.format("curl --silent --show-error --noproxy '*'  http://%s:%s/api/v1/query?query=%s",
+            K8S_NODEPORT_HOST, nodeportserver, searchKey);
+
+    logger.info("Executing Curl cmd {0}", curlCmd);
+    logger.info("Checking searchKey: {0}", searchKey);
+    logger.info(" expected Value {0} ", expectedVal);
+
+
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Check prometheus metric {0} against expected {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                searchKey,
+                expectedVal,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(searchForKey(curlCmd, expectedVal));
+  }
+
+  /**
+   * Check output of the command against expected output.
+   *
+   * @param cmd command
+   * @param searchKey expected response from the command
+   * @return true if the command succeeds
+   */
+  public static boolean execCommandCheckResponse(String cmd, String searchKey) {
+    CommandParams params = Command
+        .defaultCommandParams()
+        .command(cmd)
+        .saveResults(true)
+        .redirect(false)
+        .verbose(true);
+    return Command.withParams(params).executeAndVerify(searchKey);
+  }
+
+  /**
+   * Check if executed command contains expected output.
+   *
+   * @param cmd   command to execute
+   * @param searchKey expected output
+   * @return true if the output matches searchKey otherwise false
+   */
+  private static Callable<Boolean> searchForKey(String cmd, String searchKey) {
+    return () -> execCommandCheckResponse(cmd, searchKey);
+  }
+
+  /**
+   * Create mii image with monitoring exporter webapp.
+   */
+  private static String createAndVerifyMiiImage(String monexpAppDir, String modelFilePath) {
```

Comment:
Modify the method name createAndVerifyMiiImage() to not confuse with the utility method createMiiImageAndVerify(()

Reply:
done